### PR TITLE
Add backend calls for OS Upgrade step to Upgrade UI

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

**Comment by rhafer** [comment]
needs  rebase once https://github.com/crowbar/crowbar-core/pull/204 is merged

File: crowbar_framework/app/views/setups/show.html.haml [hunk 1]
```diff
@@ -41,22 +41,33 @@
...
         .panel-body
           %dl.dl-horizontal
+            Wait for all nodes to come back to "ready".
             %dt Upgrading node:
             %dd 134
             %dt Nodes left to update:
             %dd 89
           %p The button below should be class btn-default until the nodes are succesfully upgraded, then is should have the class btn-success
-          = button_to "Continue Upgrade Process", nodes_setup_path, class: "btn btn-primary center-block", method: :get
+          = button_to "Continue Upgrade Process", setup_path, class: "btn btn-primary center-block", method: :get
```
Comment (jsuchome) [inline crowbar_framework/app/views/setups/show.html.haml]: Shouldn't here be `nodes_os_upgrade_setup_path`?

File: crowbar_framework/app/views/setups/show.html.haml [hunk 2]
```diff
@@ -41,22 +41,33 @@
           %button.btn.btn-success.center-block{:type => "button"} Continue Upgrade Process
 %section
   .container
-    %p Step 7:
+    %p Step 8:
```
Comment (jsuchome) [inline crowbar_framework/app/views/setups/show.html.haml]: Now you have Step 8 twice and no Step 7...

Comment (rhafer) [inline crowbar_framework/app/views/setups/show.html.haml]: Isn't 7 some unlucky number anyway? :)

Comment (rhafer) [inline crowbar_framework/app/views/setups/show.html.haml]: fixed

**Comment by jsuchome** [comment]
+1

**Comment by rhafer** [comment]
rebased

**Comment by dirkmueller** [comment]
+1

**Pull request merged**